crates/routes/src/images.rs

```diff
@@ -103,6 +106,7 @@ async fn image(
   client: web::Data<Client>,
 ) -> Result<HttpResponse, Error> {
   let mut client_req = client.request_from(url, req.head());
+  client_req.headers_mut().remove(ACCEPT_ENCODING);
```

Comment:
I'll test this today. I coulda swore I tried and failed when doing this on my own system.